src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/questionWithResponses.tag

```diff
@@ -56,9 +56,8 @@
           </c:forEach>
         </ul>
       </c:if>
-
       <c:if test="${not isRecipientNameHidden}">
-        <div class="col-sm-12 form-inline mobile-align-left">
+        <div class="evalueeLabel-${questionWithResponses.question.qnIndx} form-inline mobile-align-left">
```

Comment:
i do add the instructions outside this block atm. It looks something like this:
![image](https://user-images.githubusercontent.com/11634699/40718749-1bfcffea-6409-11e8-8a00-4165fadfa6cc.png)

As you can see the instructions are added outside the `evalueeLabel` div. The problem is that I need that div as a hook  to add my elements before it (using `before` funciton). If recipient names are hidden the div is not there and hence the instructions are not displayed. 

That why I think I will simply add an empty div before that if condition of the form:
`<div class="constraints-${questionWithResponses.question.qnIndx}"></div>`
and use it as the new hook. Does that make sense ?

Reply:
uhmmm hv u considered using a div just to contain the instructions? using something like [prepend](http://api.jquery.com/prepend/) so that this instruction div will be independent of this if condition?

Reply:
yes that's what i have done in my latest commit [c4f442b](8836/commits/c4f442b74c3e604637d9b66eeaa4130fca32672e)